$$\require{mhchem}$$

### [HOME](https://wangyu16.github.io/GeneralChemistry/)

# <center>General Chemistry Study Guide</center>

### <center>Chapter 14. Chemical Kinetics</center>

---

*<center>Yu Wang</center>*

##### <b>OpenStax</b> 12 Kinetics. <b>Brown</b> 14 Chemical Kinetics. #####

## 1. Rate of Reaction and Rate Law
### Rate of Reaction
For the reaction

$$a\ce{A} +b\ce{B} \ce{->} c\ce{C} +d\ce{D}$$

The reaction rate is 

$$\begin{align}rate=-\frac{1}{a}\frac{\Delta [\ce{A}]}{\Delta t}=-\frac{1}{b}\frac{\Delta [\ce{B}]}{\Delta t}=\frac{1}{c}\frac{\Delta [\ce{C}]}{\Delta t}=\frac{1}{d}\frac{\Delta [\ce{D}]}{\Delta t}\end{align}$$

<div class=example> 
    
<b>Example:</b> Consider the reaction

$$\ce{4PH3(g) -> P4(g) + 6H2(g)}$$

Suppose that, at a particular moment during the reaction, molecular hydrogen is being formed at the rate of $0.078\ \text{M/s}$. (a) At what rate is $\ce{P4}$ being formed? (b) At what rate is $\ce{PH3}$ reacting?<br>

<b>Answer:</b> <br>
    
Step 1. Write the reaction rate expressions from the reaction<br>

$$rate=-\frac{1}{4}\frac{\Delta[\ce{PH3}]}{\Delta t}=\frac{\Delta[\ce{P4}]}{\Delta t}=\frac{1}{6}\frac{\Delta[\ce{H2}]}{\Delta t}$$

Step 2. Translate the information "molecular hydrogen is being formed at the rate of $0.078\ \text{M/s}$" to a mathematic expression

$$\frac{\Delta[\ce{H2}]}{\Delta t}=0.078\ \text{M/s}$$

Step 3. Calculate $\frac{\Delta[\ce{P4}]}{\Delta t}$ and $\frac{\Delta[\ce{PH3}]}{\Delta t}$. 

$$\frac{\Delta[\ce{P4}]}{\Delta t}=\frac{1}{6}\frac{\Delta[\ce{H2}]}{\Delta t}=\frac{1}{6}\times 0.078\ \text{M/s}=0.013\ \text{M/s}$$

$$\frac{\Delta[\ce{PH3}]}{\Delta t}=-\frac{4}{6}\frac{\Delta[\ce{H2}]}{\Delta t}=-\frac{4}{6}\times 0.078\ \text{M/s}=-0.052\ \text{M/s}$$   
    
</div>    

> **Requirements** 
> 1. Write the rate expressions for given reactions in terms of the disappearance of the reactants and the appearance of the products. Or write a balanced equation for a reaction whose rate is given. 
> 2. Knowing the consuming rate of a reactant (or the generating rate of a product), tell the rate of the consentration changes of other reactants and products.

### Rate Law
The **rate law** expresses the relationship of the rate of a reaction to the rate constant ($k$) and the concentrations of the reactants raised to some powers.

$$a\ce{A} +b\ce{B} \ce{->} c\ce{C} +d\ce{D}$$

$$\begin{align}rate=k[\ce{A}]^x[\ce{B}]^y\end{align}$$

* Reaction is $x$th order in A
* Reaction is $y$th order in B
* Reaction is $(x + y)$th order overall

$x$ and $y$ are NOT related with $a$ and $b$. The values of $x$ and $y$ are determined experimentally. 

$k$ depends on the nature of the reactants and the temperature, but does not depend on the concentrations of the reactants.

The unit of $k$ is to keep the unit balanced on both sides.


<div class=example> 
    
<b>Example:</b> Using the initial rates method and the experimental data, determine the rate law and the value of the rate constant for this reaction:
    $$\ce{2NO + Cl2 -> 2NOCl}$$
    
<table>
<thead>
  <tr>
    <th>Trial</th>
    <th>[$\ce{NO}$] (mol/L)</th>
      <th>[$\ce{Cl2}$] (mol/L)</th>
    <th>$-\frac{\Delta[\ce{NO}]}{\Delta t}$ (mol L$^{-1}$ s$^{-1}$)</th>
  </tr>
</thead>
<tbody>
  <tr>
    <td>1</td>
    <td>0.10</td>
    <td>0.10</td>
    <td>0.00300</td>
  </tr>
  <tr>
    <td>2</td>
    <td>0.10</td>
    <td>0.15</td>
    <td>0.00450</td>
  </tr>
  <tr>
    <td>3</td>
    <td>0.15</td>
    <td>0.10</td>
    <td>0.00675</td>  
  </tr>
</tbody>
</table>   
<br>    
<b>Answer:</b> <br>

The reaction rate could be expressed as:
    
\begin{align*}
    \text{rate}=k[\ce{NO}]^x[\ce{Cl2}]^y
\end{align*}
    
Plug in the experimental results
    
\begin{align*}
    \text{rate}_1=k\times 0.10^x\times0.10^y=0.00300\\
    \text{rate}_2=k\times 0.10^x\times0.15^y=0.00450\\
    \text{rate}_3=k\times 0.15^x\times0.10^y=0.00675\\
\end{align*}
    
Compare rate$_1$ and rate$_2$

\begin{align*}
    \frac{\text{rate}_2}{\text{rate}_1}=\left(\frac{0.15}{0.10}\right)^y=\frac{0.00450}{0.00300}   
\end{align*}    
    
Thus $y=1$
    
Compare rate$_1$ and rate$_3$
    
\begin{align*}
    \frac{\text{rate}_3}{\text{rate}_1}=\left(\frac{0.15}{0.10}\right)^x=\frac{0.00675}{0.00300}   
\end{align*}    
    
Thus $x=2$    
</div>

> **Requirements** 
> 1. Determine the values of $x$, $y$ and $k$ from experimental data.

## 2. Reaction Orders
### First-Order Reactions
A **first-order reaction** is a reaction whose rate depends on the reactant concentration raised to the first power. 

$$\ce{A}\ce{->} \ce{B} $$  
$$\begin{align}
& rate = k[\ce{A}] \\
& rate =-\frac{\Delta[\ce{A}]}{\Delta t} \\
& k[\ce{A}] =-\frac{\Delta[\ce{A}]}{\Delta t} \\
& k=\frac{rate}{[\ce{A}]}=\frac{\text{M/s}}{\text{M}}=\text{1/s}\ \text{or}\ \text{s}^{-1} \\
\end{align}$$
  
The concentration of $\ce{A}$ and time $t$ can be correlated by the following equations.

$$\begin{align}
& \ln\frac{[\ce{A}]_t}{[\ce{A}]_0}=-kt \\
& \ln [\ce{A}]_t=-kt+ \ln [\ce{A}]_0
\end{align}$$

where $[\ce{A}]_t$ is the concentration of $\ce{A}$ at time $t$; $[\ce{A}]_0$ is the concentration of $\ce{A}$ at time $0$.

**Half-life**  
The *half-life* of a reaction, $t_{1/2}$, is the time requied for the concentration of a reactant to decrease to half of its initial concentration.  
Half-life of a first-order reaction is  
$$\begin{align}t_{1/2}=\frac{\ln 2}{k}\end{align}$$
 

<div class=example> 
    
<b>Example:</b> The rate constant for the first-order decomposition of cyclobutane, $\ce{C4H8}$ at 500 °C is $9.2\times10^{−3}\ \text{s}^{−1}$. (a) Calculate the half life of cyclobutane. (b) If the initial amount of cyclobutane is 1 mol, calculate the amount of cyclobutane remaining after 3 minutes reaction. <br>

<b>Answer:</b> <br>
    
(a) \begin{align*}
t_{1/2} & = \frac{\ln 2}{k}\\
& = \frac{\ln 2}{9.2\times 10^{-3}\ \text{s}^{-1}}\\
& = 75\ \text{s}
\end{align*}

(b) \begin{align*}
& \ln\left(\frac{[\ce{C4H8}]_0}{[\ce{C4H8}]}\right)=kt=9.2\times10^{−3}\ \text{s}^{−1}\times 180\ \text{s}=1.6\\
& \frac{[\ce{C4H8}]_0}{[\ce{C4H8}]}=4.9\\
& [\ce{C4H8}]=0.20\ \text{mol}
\end{align*}
</div>

> **Requirements**  
> 1. Calculate half-life of a first-order reaction from known rate constant. 
> 2. Calculate $k$ from the plot of $\ln [\ce{A}]$ vs $t$. 
> 2. Knowing $k$ and $[\ce{A}]_0$, calculate the concentration of $\ce{A}$ after time $t$; or knowing $k$, $[\ce{A}]_0$ and $[\ce{A}]_t$, calculate the time $t$.

### Second-Order Reactions

A **second-order reaction** is a reaction whose rate depends on the concentration of one reactant raised to the second power or on the concentrations of two different reactants, each raised to the first power. 

$$\ce{A}\ce{->} \ce{B} \\$$
$$\begin{align}
& rate = k[\ce{A}]^2 \\
& rate =-\frac{\Delta[\ce{A}]}{\Delta t} \\
& k[\ce{A}]^2 =-\frac{\Delta[\ce{A}]}{\Delta t} \\
& k=\frac{rate}{[\ce{A}]^2}=\frac{\text{M/s}}{\text{M}^2}=\text{1/M}\cdot\text{s}\ \text{or}\ \text{M}^{-1}\text{s}^{-1} \\
\end{align}$$

The concentration of $\ce{A}$ and time $t$ can be correlated by the following equation.

$$\begin{align}
\frac{1}{[\ce{A}]_t}=\frac{1}{[\ce{A}]_0}+kt
\end{align}$$

where $[\ce{A}]_t$ is the concentration of $\ce{A}$ at time $t$; $[\ce{A}]_0$ is the concentration of $\ce{A}$ at time $0$.

The half-life of a second-order reaction is 

$$\begin{align}
t_{1/2}=\frac{1}{k[\ce{A}]_0}
\end{align}$$

<div class=example> 
    
<b>Example:</b> The reaction of butadiene gas to yield $\ce{C8H12}$ gas is a second order reaction with a rate constant $5.76\times10^{-2}\ \text{L mol}^{-1}\ \text{min}^{-1}$. If the initial concentration of butadiene is $0.200\ \text{mol L}^{-1}$, what is the concentration after 10.0 min?<br>

$$\ce{2C4H6 -> C8H12}$$

<b>Answer:</b> <br>    
\begin{align*}
& \frac{1}{[\ce{C4H6}]_t}\\ 
& =\frac{1}{[\ce{C4H6}]_0}+kt\\
& = \frac{1}{0.200\ \text{mol L}^{-1}} + 5.76\times10^{-2}\ \text{L mol}^{-1}\ \text{min}^{-1}\times 10.0\ \text{min}\\
& = 5.58\ \text{L mol}^{-1}\\
& [\ce{C4H6}]_t = 1.79\times 10^{-1}\ \text{mol L}^{-1}
\end{align*}
</div>

> **Requirements**  
> 1. Calculate  $k$  from the plot of  $\frac{1}{[\ce{A}]}$  vs  $t$.
> 2. Calculate half-life of a second-order reaction from known rate constant and initial concentration. 
> 3. Knowing $k$ and $[\ce{A}]_0$, calculate the concentration of $\ce{A}$ after time $t$; or knowing $k$, $[\ce{A}]_0$ and $[\ce{A}]_t$, calculate the time $t$. 

### Zero-Order Reactions

$$\ce{A}\ce{->} \ce{B} \\$$
$$\begin{align}
& rate = k[\ce{A}]^0  = k \\
& rate =-\frac{\Delta[\ce{A}]}{\Delta t} \\
& k =-\frac{\Delta[\ce{A}]}{\Delta t} \\
& k=\frac{rate}{[\ce{A}]^0}=\text{M/s}\\
\end{align}$$

The concentration of $\ce{A}$ and time $t$ can be correlated by the following equation.

$$\begin{align}
[\ce{A}]_t=[\ce{A}]_0-kt
\end{align}$$

where $[\ce{A}]_t$ is the concentration of $\ce{A}$ at time $t$; $[\ce{A}]_0$ is the concentration of $\ce{A}$ at time $0$.

The half-life of a second-order reaction is 

$$\begin{align}
t_{1/2}=\frac{[\ce{A}]_0}{2k}
\end{align}$$

### Summary of Reaction Orders


|                        | Zero-Order                      | First-Order                    | Second-Order                                 |
|------------------------|---------------------------------|--------------------------------|----------------------------------------------|
| rate law               | $\text{rate}=k$                 | $\text{rate}=k[\ce{A}]$        | $\text{rate}=k[\ce{A}]^2$                    |
| units of rate constant | $\text{M s}^{-1}$               | $\text{s}^{-1}$                | $\text{M}^{-1}\text{ s}^{-1}$                |
| integrated rate law    | $[\ce{A}]=[\ce{A}]_0-kt$        | $\ln[\ce{A}]=\ln[\ce{A}]_0+kt$ | $\frac{1}{[\ce{A}]}=\frac{1}{[\ce{A}]_0}+kt$ |
| half-life              | $t_{1/2}=\frac{[\ce{A}]_0}{2k}$ | $t_{1/2}=\frac{\ln 2}{k}$      | $t_{1/2}=\frac{1}{[\ce{A}]_0 k}$             |

> **Requirements**  
> 1. Understand what is a zero-order reaction. 

## 3. Temperature Effect
**Collision Theory**  
1. Reaction rate is proportional to the frequency of molecular collisions. This relationship explains the dependence of reaction rate on concentration.
2. Not all collisions result in reactions. To react, the colliding molecules must have a total kinetic energy equal to or greater than the **activation energy** ($E_a$). 

**Activiation Energy** is the minimum amount of energy required to initiate a chemical reaction.  
**Transition State** refers to the species temporarily formed by the reactant molecules as a result of the collision before they form the product. 

**Arrhenius Equation**  
$$\begin{align}
& k=Ae^{-E_a/RT} \\
& \ln k = \ln A - \frac{E_a}{RT} \\
\end{align}$$

Alternate form of Arrhenius Equation
$$\ln\frac{k_1}{k_2}=\frac{E_a}{R}\left(\frac{1}{T_2}-\frac{1}{T_1}\right)$$

<div class=example> 
    
<b>Example:</b> The rate constant for the decomposition of acetaldehyde, $\ce{CH3CHO}$, to methane, $\ce{CH4}$, and carbon monoxide, $\ce{CO}$, in the gas phase is $1.1\times10^{−2}\ \text{L mol}^{−1}\ \text{s}^{−1}$ at 703 K and $4.95\ \text{L mol}^{−1}\ \text{s}^{−1}$ at 865 K. Determine the activation energy for this decomposition.<br>

<b>Answer:</b> <br>
Plug in the numbers to the alternate form of Arrhenius Equation

$$\ln\frac{1.1\times10^{−2}\ \text{L mol}^{−1}\ \text{s}^{−1}}{4.95\ \text{L mol}^{−1}\ \text{s}^{−1}}\\=\frac{E_a}{8.314\ \text{J mol}^{-1}\ \text{K}^{-1}}\left(\frac{1}{865\ \text{K}}-\frac{1}{703\ \text{K}}\right)$$

Solve the equation and get the value of $E_a$ 

$$E_a=1.90\times10^5\ \text{J mol}^{-1}$$

</div>

> **Requirements**  
> 1. Understand collision theory; know what is activation energy and what is transition state.
> 2. Calculate rate constans at different temperatures using Arrhenius Equation.

In [3]:
# Knowing k at one temperature, calculate k at another temperature. Ea is given.
from numpy import*
Ea = 110 # in kJ/mol
T1=27 # in Celsius
T2=42 # in Celsius
k1=3.27e-6
k2=k1/exp(Ea*1000/8.314*(1/(T2+273)-1/(T1+273)))
k2

2.6706322527649514e-05

In [7]:
# Knowing A, Ea and T, calculate k
A = 7500000
Ea = 33.1 # in kJ/mol
T=25 # in Celsius
k=A*exp(-Ea/0.008314/(T+273))
k

11.829065218689623

## 4. Reaction Mechanisms

**Elementary Reactions** are a series of simple reactions that represent the progress of the overall reaction at the molecular levle.  

**Reaction Mechanism** is the sequence of elementary reactions that leads to product formation.  

An Example from the Textbook:  
The overall reaction  
$$\ce{2NO(g) + O2(g) -> 2NO2(g)}$$  
is actually a sum of two elementary reactions  
$$\ce{NO +NO -> N2O2}$$  
$$\ce{N2O2 + O2 -> 2NO2}$$  
*The sum of the elementary steps must give the overall balanced equation for the reaction. *

**Intermediats** (are not the same as transition states) are formed in early elementary steps and consumed in later elementary steps. Thus intermediats do not appear in the overall ballanced equation.  

**molecularity of a reaction** is the number of molecules reacting in an elementary step, such as unimolecular reaction, bimolecular reaction and termolecular reaction. 

**Rate Laws of Elementary Reactions**  
$$a\ce{A} +b\ce{B} \ce{->} c\ce{C} +d\ce{D}$$  
If this is an elementary reaction, then  
$$\begin{align}rate=k[\ce{A}]^a[\ce{B}]^b\end{align}$$

Since an elementary reaction usually involves only one or two molecules, there are only a few possible types of elementary reactions, which are:

\begin{align*}
\ce{A -> B}\qquad & \text{rate}=k[\ce{A}]\\
\ce{2A -> B}\qquad & \text{rate}=k[\ce{A}]^2\\
\ce{A + B -> C}\qquad & \text{rate}=k[\ce{A}][\ce{B}]
\end{align*}

Termolecular reaction is possible, but is rare. Kinetics of termolecular reactions is not discussed here.

**Rate-determining step** is the slowest step in the sequence of steps leading to product formation. 

*The rate-determining step should predict the same rate law as is determined experimentally. *

**Potential energy profile for a multi-step reaction**  
* Compare the potential energy of reactants and products to get whether the overall reaction is exothermic or endothermic. 
* The maxima are the transition states; and the number of maxima tells the number of elementary steps.
* The minima are intermediates (not including the reactions and the products). 
* Activation energies are the defference of potential energies between reactants (or intermediates) and transition states. 
* The rate-determining step corresponds to the largest activation energy. 


<div class=example> 
    
<b>Example:</b> The two-step mechanism below has been proposed for a reaction between nitrogen dioxide and carbon mononoxide:
\begin{align*}
& \text{(1)}\quad\ce{2NO2(g) -> NO3(g) + NO(g)}\qquad\text{slow}\\
& \text{(2)}\quad\ce{NO3(g) + CO(g) -> NO2(g) + CO2(g)}\qquad\text{fast} 
\end{align*}    
What is the overall reaction? What is the intermediate? What is the overall rate law?    

<b>Answer:</b> <br>
    
The overall reaction is the sum of the two steps:
    
\begin{align*}
& \ce{2NO2(g) + NO3(g) + CO(g)\\
& -> NO3(g) + NO(g) + NO2(g) + CO2(g)}    
\end{align*} 
    
This need to be simplified to:

\begin{align*}
\ce{NO2(g) + CO(g) -> NO(g) + CO2(g)}    
\end{align*}     

The intermediate is $\ce{NO3}$.
    
The overall rate law equals to the slow step, which is $\text{rate}=k[\ce{NO2}]^2$.    
</div>

> **Requirements**  
> 1. Given elementary steps, identify the intermediates and write the overall balanced reaction equation. 
> 2. Knowing the rate-determining step of a reaction, tell the rate law of the overall reaction. Or knowing the rate law of a reaction and the elementary steps, tell which is the rate-determining step. 
> 3. Given the potential energy profile of a multi-step reaction, tell whether the overal reaction is exothermic or endothermic, the number of elementary steps, the intermediate(s), the transition states, the activation energies of each step, the rate-determining step. 

## 5. Catalysis
**Catalyst** is a substance that increases the rate of a chemical reaction without itself undergoing any permanent chemical change. A catalyst lowers the activation energy for both the forward and reverse reactions.

Three types of catalysis: heterogeneous catalysis, homogeneous catalysis, and enzyme catalysis.



> **Requirements**
> 1. Know what is a catalyst and its role in a reaction. 
> 2. Know the three types of catalysis. 

<div class=practice>
    
<b>Practice</b> <br>

1. For the elementary reaction shown below, which one of the following statements can be rightly assumed?
    $$\ce{A + B -> C + D}$$
A. $\text{rate}=k[\ce{A}][\ce{B}]$<br>
B. $\text{rate}=k[\ce{A}]^2[\ce{B}]$<br>
C. $\text{rate}=k[\ce{A}]^2[\ce{B}]^2$<br>
D. $\text{rate}=k[\ce{A}][\ce{B}]^2$<br>
E. Rate law for this reaction can not be determined based on the available information. <br><br>

2. What is the correct balanced equation for the following rate expression?
$$\text{rate}=-\frac{\Delta[\ce{SO2}]}{\Delta t}=-\frac{1}{3}\frac{\Delta[\ce{CO}]}{\Delta t}=\frac{1}{2}\frac{\Delta[\ce{CO2}]}{\Delta t}=\frac{\Delta[\ce{COS}]}{\Delta t}$$    
A. $\ce{1/2CO2 + COS -> SO2 + 1/3CO}$<br>
B. $\ce{2CO2 + COS -> SO2 + 3CO}$<br>
C. $\ce{SO2 + 1/3CO -> 1/2CO2 + COS}$<br>
D. $\ce{SO2 + 3CO -> 2CO2 + COS}$<br>
E. The balanced equation cannot be determined based on the available information.<br><br>

3. Use the following data to determine the rate law for the reaction shown below.
    $$\ce{2NO(g) + 2H2(g) -> N2(g) + 2H2O(g)}$$
<table>
<thead>
  <tr>
    <th>Trial</th>
    <th>[$\ce{NO}$] (mol/L)</th>
    <th>[$\ce{H2}$] (mol/L)</th>
    <th>$-\frac{\Delta[\ce{NO}]}{\Delta t}$ (mol L$^{-1}$ s$^{-1}$)</th>
  </tr>
</thead>
<tbody>
  <tr>
    <td>1</td>
    <td>$5.0\times10^{-3}$</td>
    <td>$2.0\times10^{-3}$</td>
    <td>$1.25\times10^{-5}$</td>
  </tr>
  <tr>
    <td>2</td>
    <td>$10.0\times10^{-3}$</td>
    <td>$2.0\times10^{-3}$</td>
    <td>$5.0\times10^{-5}$</td>
  </tr>
  <tr>
    <td>3</td>
    <td>$10.0\times10^{-3}$</td>
    <td>$4.0\times10^{-3}$</td>
    <td>$10.0\times10^{-5}$</td>
  </tr>
</tbody>
</table>

A. $\text{rate}=k[\ce{NO}][\ce{H2}]$<br>
B. $\text{rate}=k[\ce{NO}]^2[\ce{H2}]$<br>
C. $\text{rate}=k[\ce{NO}][\ce{H2}]^2$<br>
D. $\text{rate}=k[\ce{NO}]^2[\ce{H2}]^2$<br>
E. $\text{rate}=k[\ce{NO}]^2[\ce{H2}]^4$<br><br>

4. The two-step mechanism below has been proposed for a reaction between nitrogen dioxide and carbon mononoxide:
\begin{align*}
& \text{(1)}\quad\ce{N2O -> N2 + O}\\
& \text{(2)}\quad\ce{N2O + O -> N2 + O2} 
\end{align*}    
What is the overall reaction? What is the intermediate? If the rate law is $\text{rate}=k[\ce{N2O}]$, which step is the rate limiting step?<br><br>

5. For the reaction $\ce{BrO3- + 5Br- + 6H+ -> 3Br2 + 3H2O}$ at a particular time, the molecular $\ce{H2O}$ is being formed at the rate of $1.5\times 10^{-2}$ M/s. What rate is $\ce{H+}$ reacting? (Answer: $3.0\times 10^{-2}$ M/s)<br><br>

6. The reaction $\ce{A + 2B -> C}$ has the rate law $rate=k[\ce{A}][\ce{B}]^3$. If the concentration of $\ce{B}$ is doubled while that of $\ce{A}$ is unchanged, by what factor will the rate of reaction increase? (Answer: 8)<br><br>

7. The activation energy for the following first-order reaction is 110 kJ/mol. 
$$\ce{N2O5(g) <=> 2NO2(g) + \frac{1}{2}O2(g)}$$	
The value of the rate constant, $k$, is $3.27\times 10^{-6}\ \text{s}^{-1}$ at 27 $^\circ$C. What is the value of $k$ at 42 $^\circ$C? (Answer: $2.67\times 10^{-5}$ s$^{-1}$)<br><br>

8. The initial concentration of the reactant A is 0.58 M in a second-order reaction $\ce{A -> B}$. After 28 min, the concentration of A decreased to 0.16 M. What is the rate constant of the reaction? How long would it take in minutes for the reaction to be $90\%$ complete starting from 0.58 M of A? (Answer: $k = 0.162$ M$^{-1}$min$^{-1}$, $t = 95.8$ min)  
</div>    

---

In [1]:
from IPython.core.display import HTML
def css_styling():
    styles = open("custom.css", "r").read()
    return HTML(styles)
css_styling()